src/plugins/kibana_usage_collection/server/collectors/ops_stats/ops_stats_collector.ts

```diff
@@ -12,8 +12,7 @@ import moment from 'moment';
 import { OpsMetrics } from 'kibana/server';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
 import { KIBANA_STATS_TYPE } from '../../../common/constants';
-
-interface OpsStatsMetrics extends Omit<OpsMetrics, 'response_times' | 'collected_at'> {
+interface OpsStatsMetricsUsage {
```

Comment:
Please, mind that we've removed the `extends Omit<OpsMetrics, 'response_times' | 'collected_at'> {` bit! That's a lot of fields we are missing.

Reply:
I was working towards what we return from the `fetch` and must have interpreted the return data incorrectly.
We have 2 options here:
1. Revert the changes and don't declare the schema or
2. Report all the fields.